.github/workflows/test.yml

```diff
@@ -5,35 +5,35 @@ on:
       - "*"
 jobs:
   lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     name: Lint
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: "14"
+          node-version: "16"
```

Comment:
Grab the node version like [this](https://github.com/dltxio/nestjs-template/blob/main/.github/workflows/build-project.yml)

Reply:
👍